### PR TITLE
Add Spanish language

### PR DESCRIPTION
**Pull request by MrBlasyMSK**

### CODE REVIEW

**Comment by supercrafter333** [comment]
Thank you for translate. I'll review this later but it looks well.

**Comment by supercrafter333** [comment]
Your translation should be compatible with theSpawn v1.7.0-dev. theSpawn v1.6.2 will be outdated soon, theSpawn v1.7.0-dev is also on poggit now.

File: resources/Languages/es_ES.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+#formas
+#   warps
+form-warp-menu-title: "Warps"
+form-warp-menu-content: "Por favor elecciona un warp."
+form-warp-menu-warpButton: "§b{warp}§r{line}§7clic para teletransportarse"
+#   setwarp
+form-setWarp-menu-title: "Establecer un warp"
+form-setWarp-menu-content: "Aquí puedes establecer un nuevo warp."
+form-setWarp-menu-inputNameDescription: "Nombre del nuevo warp"
+form-setWarp-menu-inputPermDescription: "Permiso del nuevo warp (opcional)"
```
Comment (supercrafter333) [inline resources/Languages/es_ES.yml:86]: In theSpawn v1.7.0-dev (and newer) this part won't work. 
Please use:
```yaml
form-setWarp-menu-togglePermDescription: "Should the new warp have a permission? (optional)"
form-setWarp-menu-inputIconPathDescription: "Icon-path of the new warp (optional)"
```
[line 88 and 89](https://github.com/supercrafter333/theSpawn/commit/ef17d314a07d6b10b638fdbe57c90be2da6897cb#diff-0b934b18ed444bc07da12626e22dc539ba047e86c8b8a05c987df1bfb3926a28R88-R89)

File: resources/Languages/es_ES.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+#mensajes de casas
+#usos: {home}
+home-teleport: "§aHas sido teletransportado con éxito a tu casa §b{home}§a!"
+home-not-exists: "§cTu casa §b{home}§c no existe!"
+home-set: "§aHas guardado con éxito tu hogar §b{home}§a!"
+home-deleted: "§aHas eliminado con éxito tu casa §b{home}§a!"
+home-already-exists: "§cTu casa §b{home}§c ya existe!"
+#usos: {homelist}
+homelist: "§eTus casas: §7{homelist}"
+
```
Comment (supercrafter333) [inline resources/Languages/es_ES.yml:40]: In theSpawn v1.7.0-dev (and newer) there is a new message:
```yaml
#uses: {max-homes}
highest-home-count-reached: "§cYou've already reached the highest count of §r{max-homes}§c homes!"
```
[line 40 and 41](https://github.com/supercrafter333/theSpawn/commit/ef17d314a07d6b10b638fdbe57c90be2da6897cb#diff-0b934b18ed444bc07da12626e22dc539ba047e86c8b8a05c987df1bfb3926a28R40-R41)

**Review by supercrafter333 — CHANGES_REQUESTED** [review]

**Comment by MrBlasyMSK** [comment]
Is everything correct now?

**Comment by supercrafter333** [comment]
[That](https://github.com/supercrafter333/theSpawn/pull/34/files/f3102ecbc0075a02bcbd356c3f1fc0019be78592#r826818894) isn't resolved.

**Comment by MrBlasyMSK** [comment]
> [That](https://github.com/supercrafter333/theSpawn/pull/34/files/f3102ecbc0075a02bcbd356c3f1fc0019be78592#r826818894) isn't resolved.

Its at the line 88 and 89 in https://github.com/MrBlasyMSK/theSpawn/blob/patch-1/resources/Languages/es_ES.yml
```yml
form-setWarp-menu-togglePermDescription: "¿Debe el nuevo warp tener un permiso? (opcional)"
form-setWarp-menu-inputIconPathDescription: "Icono del nuevo warp (opcional)"
```

**Pull request merged**